### PR TITLE
bump and build 2.4.4

### PR DESCRIPTION
**Pull request by paulfalgout**

### CODE REVIEW

**Comment by jasonLaster** [comment]
:+1:

**Comment by paulfalgout** [comment]
Updated to include https://github.com/marionettejs/backbone.marionette/pull/2826

File: changelog.md [hunk 1]
```diff
@@ -18,7 +32,6 @@
 
 #### Misc
 
-* `LayoutView#showChildView` new `options` argument passed to underlying `Region#show` to enable full `show` functionality.
```
Comment (samccone) [inline changelog.md]: ?

Comment (paulfalgout) [inline changelog.md]: Turns out that didn't get in `2.4.3` though the PR was labeled that way it was only merged into `next`.  People were asking about it on gitter, so it was cherry-picked on to patch for `2.4.4`

Comment (samccone) [inline changelog.md]: heh

**Comment by samccone** [comment]
:+1: LGTM

**Pull request merged**